### PR TITLE
Cloud copy updates

### PR DESCRIPTION
**Pull request by carkod**
## Done

Copy updates for /cloud/ pages

## QA

- Check https://ubuntu-com-12532.demos.haus/cloud/hybrid-cloud against [copy docs](https://docs.google.com/document/d/1oIggNSXR5sfl0Ab35x2NIL7I858I7QdPCUXeQI8IIPc/edit#)
- Check https://ubuntu-com-12532.demos.haus//cloud/multi-cloud against [copy docs](https://docs.google.com/document/d/1nWsDaRiUzaupjKDzu1ZqjlmxHGv2HCgW8oBBEacYFLs/edit#heading=h.6gx8ccowcd3a)
- Check https://ubuntu-com-12532.demos.haus/cloud/private-cloud against [copy docs](https://docs.google.com/document/d/19g24YRYzF5dBpUKI1QzTG7fXKj1z0OvwgFll69dFoOs/edit#heading=h.evj3u5y98j9v)

## Issue / Card

Closes https://warthogs.atlassian.net/browse/WD-1322?atlOrigin=eyJpIjoiYTczMGNmNGY3Yjc0NDc0MGE4YTBmZTgzNzA5ZDMwZWIiLCJwIjoiaiJ9
Closes https://warthogs.atlassian.net/browse/WD-1321?atlOrigin=eyJpIjoiODA4MWZjMjg0YTFiNDA5YWE4N2NkYmFkMjE2ZmE4MTkiLCJwIjoiaiJ9
Closes https://warthogs.atlassian.net/browse/WD-1320?atlOrigin=eyJpIjoiZTYwNzEwZTUwOWIwNDUwYWE5YjJhYzc3NjQ3NGZkNzUiLCJwIjoiaiJ9
Closes https://warthogs.atlassian.net/browse/WD-1320?atlOrigin=eyJpIjoiNzMyODNlYjE0MWY2NDg2YThmOWM0NWQ5MmRlODVkNjgiLCJwIjoiaiJ9

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/cloud/hybrid-cloud.html [hunk 1]
```diff
@@ -379,29 +379,41 @@ <h2 class="u-sv2">All you need for your hybrid cloud</h2>
...
+          loading="lazy"
+          ) | safe
+        }}
       </div>
       <hr/>
       <div class="p-card__content">
-        <p>Charmed OpenStack is a pure upstream OpenStack distribution engineered for the best price-performance with full enterprise support for your production-grade private cloud.</p>
-        <p><a href="/openstack/features">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a></p>
+        <p>Canonical OpenStack is an enterprise cloud platform engineered for price-performance that serves as a cost-effective extension to hyperscale infrastructure.</p>
+        <p><a href="https://ubuntu.com/openstack/">Learn more about Canonical OpenStack&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/cloud/hybrid-cloud.html:397]: This link should be relative.

File: templates/cloud/multi-cloud.html [hunk 2]
```diff
@@ -11,10 +11,10 @@
   <div class="row">
     <div class="col-7">
       <h1>What is multi-cloud?</h1>
-      <p>Multi-cloud (also referred to as multi cloud or multi-cloud) is a concept that refers to using multiple clouds from more than one cloud service provider at the same time. The term is also used to refer to the simultaneous running of bare metal, virtualised and containerised workloads.</p>
+      <p>Multi-cloud (also referred to as multi cloud or multi-cloud) is a concept that refers to using multiple <a href="/engage/multi-cloud-business-guide">clouds</a> from more than one cloud service provider at the same time. The term is also used to refer to the simultaneous running of bare metal, virtualised and containerised workloads.</p>
```
Comment (anthonydillon) [inline templates/cloud/multi-cloud.html:14]: I would suggest updating the link to cover "using multiple clouds".

File: templates/cloud/multi-cloud.html [hunk 3]
```diff
@@ -359,37 +359,48 @@ <h2>How to build a multi-cloud</h2>
...
+          loading="lazy"
+          ) | safe
+        }}
       </div>
       <hr/>
       <div class="p-card__content">
-        <p>Charmed OpenStack is a pure upstream OpenStack distribution engineered for the best price-performance with full enterprise support for your production-grade private cloud.</p>
-        <p><a href="/openstack/features">Learn more about Charmed OpenStack&nbsp;&rsaquo;</a></p>
+        <p>Canonical OpenStack is an enterprise cloud platform engineered for price-performance that serves as a cost-effective extension to hyperscale infrastructure.</p>
+        <p><a href="https://ubuntu.com/openstack/">Learn more about Canonical OpenStack&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/cloud/multi-cloud.html:382]: This link should be relative.

File: templates/cloud/multi-cloud.html [hunk 4]
```diff
@@ -359,37 +359,48 @@ <h2>How to build a multi-cloud</h2>
 <section class="p-strip">
   <div class="row u-equal-height">
     <div class="col-8">
-      <h2 class="u-sv2">All you need for your multi-cloud</h2>
+      <h2 class="u-sv2">All you need for your hybrid cloud</h2>
```
Comment (anthonydillon) [inline templates/cloud/multi-cloud.html:362]: We use multi-cloud above and use hybrid cloud here. Are they the same?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments and questions in the code

**Comment by carkod** [comment]
Blocked by https://warthogs.atlassian.net/browse/WD-1315

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**